sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-binder/pom.xml

```diff
@@ -12,7 +12,12 @@
   <artifactId>azure-spring-cloud-sample-eventhubs-binder</artifactId>  
   <groupId>com.azure.spring</groupId>  
   <version>1.0.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-sample-eventhubs-binder;current} -->
-  <name>Azure Spring Cloud Sample Event Hubs Binder</name>  
+  <name>Azure Spring Cloud Sample Event Hubs Binder</name>
+
+  <properties>
```

Comment:
why adding this?

Reply:
Otherwise, test IT will not be found to execute, also need 'org.junit.jupiter' dependencies.